### PR TITLE
Added merchant redirect documentation

### PR DESCRIPTION
**Pull request by khenrichsen**
- Added api documentation for Merchant Redirect QR
- Added flow-diagrams for both QR flows
- Updated examples in the docs
- Added redoc urls for examples
- Added example QR
- Added svg+xml to accept headers
- Updated FAQ
- Updated documentation and API version
- Added fallback url to FAQ

### CODE REVIEW

File: vipps-qr-api-faq.md [hunk 1]
```diff
@@ -18,19 +16,15 @@ Document version 0.0.1.
 It contains answers to all(?) common questions about Vipps payments:
 [Vipps eCom API FAQ](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md).
 
-## Can we make our own QR code based on the Vipps deeplink URL?
-
-Technically: Yes.
+## Cant we make our own QRs for payment and redirects?
```
Comment (flinkgutt) [inline vipps-qr-api-faq.md:19]: Use `can't`, 'cant' means something different.
Or just use the original phrasing.

File: vipps-qr-api-faq.md [hunk 2]
```diff
@@ -39,7 +33,7 @@ The Vipps QR API is an addition to the
 Merchants can integrate with the Vipps eCom API for many different use cases.
 The Vipps QR API adds "just" the QR code functionality.
 
-Yes, there will be some extra API calls, but the overhead is minimal.
+Yes, there is one extra API calls, but the overhead is minimal.
```
Comment (flinkgutt) [inline vipps-qr-api-faq.md:36]: `call`, singular. Not `calls` plural.

**Comment by cloveras** [comment]
There are a lot of "url" what should be changed to "URL". Let the capitalisation begin!

File: vipps-qr-api.md [hunk 3]
```diff
@@ -60,16 +48,61 @@ For more information about how to obtain an access token and all details around
...
-# Initiate a payment with the Vipps eCom API
+## QR formats
 
-Before creating the QR code you must initiate a payment with the Vipps eCom API as described in the
-[eCom API guide](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#initiate-payment-flow-phone-and-browser).
+The QR code image will be returned as a URL in the response for both otp- and mr-QRs. Opening this url will return the image in the format (and resolution) set in the accept header. The URL to the image will look like this:
+```json
+"url":"https://qr-generator-prod-app-service.azurewebsites.net/qr-generator/v1?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...."
+```
+The URL to the images does not require any authentication, and can be shown wherever you want. Note that oneTimePayment QRs eventually will time out - trying to open the url to the otp-QR image after its expiry while will return a 404. Merchant redirect QRs can be openened forever.
```
Comment (flinkgutt) [inline vipps-qr-api.md:57]: `openened` is not correct. Replace with `opened`. (Future perfect)

File: vipps-qr-api.md [hunk 4]
```diff
@@ -112,51 +146,68 @@ Body:
...
-This will require an approval from Vipps before it is used, so we can validate
-the styling of the QR.
+The QR code, when scanned from native camera or the Vipps scanner, will take the customer straight to webpage. These QRs doesnt require the Vipps app to be installed.
 
-The `targetUrl` that points to `https://short.vipps.no` is a shortened URL
-that will redirect to the payment URL that was posted to the API.
-Since the `targetUrl` is quite short, the generated QR code is less complex and
-easier to scan efficiently.
+## Creation of merchant redirect QR
+To create a merchantRedirect QR, make a HTTP POST to:
```
Comment (flinkgutt) [inline vipps-qr-api.md:168]: HTTPS! Gotta secure it!

File: vipps-qr-api-faq.md (hunk 1 above)
Comment (khenrichsen) [reply vipps-qr-api-faq.md:19]: Or even better - just "can" in this case 😅

**Review by flinkgutt — APPROVED** [review]
LGTM 👍

**Pull request merged**